src/Standards/Generic/Sniffs/CodeAnalysis/JumbledIncrementerSniff.php

```diff
@@ -85,11 +85,11 @@ public function process(File $phpcsFile, $stackPtr)
             }
 
             $inner = $this->findIncrementers($tokens, $tokens[$start]);
-            $diff  = array_intersect($outer, $inner);
+            $commonIncrementers = array_intersect($outer, $inner);
```

Comment:
While I agree that the name `$diff`is not the best name, `$commonIncrementers` isn't either as it contains the assumption that variables will always be incremented, while they could also be decremented, or even be changed via a function call or some other calculation.

Maybe it should be called `$commonVariables` or `$varIntersection` or something ? (I'm not sure what the best name is, but the current name change, IMO, is not an improvement).

Reply:
As we talked about during our call, we won't be changing this variable name at this time and we will leave it for when the other problems you found are addressed in a separate issue.

Reply:
I will go ahead, drop the commit that changed this variable name and force push the branch.